doc/news/interface_changes/README.rst

```diff
@@ -0,0 +1,9 @@
+Interface Changes
+=================
+
+Add news fragments for interface changes (e.g. xml changes) to this directory.
+The format should be ``<JIRA TICKET>.<COMPONENT>.<EXT>``, where ``<COMPONENT>`` is the name of the component in lowercase.
+The file extension defining the markup format can either be ``rst`` or ``md``.
```

Comment:
I'm not sure if you can hotswap formats, for example the changelog is in rst and if the fragment is in md whether that would work or not.